File: Operacoes/resto.cpp
/*@resto: Resultado e resto na divisão*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    
    int n1, n2;

    cin >> n1;
    cin >> n2;

    float resto = n1 % n2;
    float quociente = n1 / n2;

    cout << quociente << " " << resto << "\n"; 
    return 0;
}

File: src/Operacoes/Resto.java
/**
 * Questão: @resto: Resultado e resto na divisão
 *
 * @author UxieGu1
 */
package Operacoes;

import java.util.Scanner;

public class Resto {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n1 = scanner.nextInt();
        int n2 = scanner.nextInt();

        int resto = n1 % n2;
        int quociente = n1 / n2;

        System.out.println(quociente + " " + resto);
        scanner.close();
    }
}
